go-controller/pkg/ovn/policy.go

```diff
@@ -1207,11 +1207,13 @@ func (oc *Controller) addNetworkPolicy(policy *knet.NetworkPolicy) error {
 	nsInfo.networkPolicies[policy.Name] = np
 	// there may have been a namespace update for ACL logging while we were creating the NP
 	// update it
-	if err := oc.setNetworkPolicyACLLoggingForNamespace(policy.Namespace, nsInfo); err != nil {
-		klog.Warningf(err.Error())
-	} else {
-		klog.Infof("Namespace %s: ACL logging setting updated to deny=%s allow=%s",
-			policy.Namespace, nsInfo.aclLogging.Deny, nsInfo.aclLogging.Allow)
+	if nsInfo.aclLogging.Allow != aclLogAllow {
```

Comment:
are you trying to just optimize this? or this is some bug fix?

Reply:
both, it's a scale bug.

Reply:
`setNetworkPolicyACLLoggingForNamespace` also updates Deny priority that could've changed too, are you excluding it here on purpose?

Reply:
What do you mean with Deny priority?

If you mean logging for default deny ACLs, yes I am ignoring those on purpose, those will be updated as part of namespace update handling.

We are only concerned about logging changes that happened while `createNetworkPolicy` was ongoing, which only ever cares about allow logging.

Reply:
thanks, now I get the idea

Reply:
This is totally not obvious unless you stare at the code and see that we hold nsinfo lock while we create the default deny ACLs. I think you need to add a comment here stating that's why its ok to only check aclLogging.Allow